File: testdata/tpl_nav_bar/expected.swift
struct ContentView: View {
    public var body: some View {
        HStack(alignment: .center, spacing: 0) {
            Text("logo")
                .font(.system(size: 26))
                .foregroundColor(Color(red: 0.05, green: 0.05, blue: 0.1).opacity(0.85))
                .frame(width: 48.0, height: 48.0)
                .padding(8.0)
                .background(Color(red: 0.98, green: 0.71, blue: 0.68))
            Spacer(minLength: 0)
            HStack(alignment: .center, spacing: 8.0) {
                Text("link-1")
                    .font(.system(size: 26))
                    .foregroundColor(Color(red: 0.05, green: 0.05, blue: 0.1).opacity(0.85))
                    .frame(width: 80.0, height: 36.0)
                    .padding(8.0)
                    .background(Color(red: 0.70, green: 0.80, blue: 0.89))
                Text("link-2")
                    .font(.system(size: 26))
                    .foregroundColor(Color(red: 0.05, green: 0.05, blue: 0.1).opacity(0.85))
                    .frame(width: 80.0, height: 36.0)
                    .padding(8.0)
                    .background(Color(red: 0.80, green: 0.92, blue: 0.77))
                Text("link-3")
                    .font(.system(size: 26))
                    .foregroundColor(Color(red: 0.05, green: 0.05, blue: 0.1).opacity(0.85))
                    .frame(width: 80.0, height: 36.0)
                    .padding(8.0)
                    .background(Color(red: 0.87, green: 0.80, blue: 0.89))
            }
            .background(Color(red: 0.11, green: 0.11, blue: 0.17))
            Spacer(minLength: 0)
            HStack(alignment: .center, spacing: 8.0) {
                Text("btn-1")
                    .font(.system(size: 26))
                    .foregroundColor(Color(red: 0.05, green: 0.05, blue: 0.1).opacity(0.85))
                    .frame(width: 36.0, height: 36.0)
                    .padding(8.0)
                    .background(Color(red: 1.00, green: 0.85, blue: 0.65))
                Text("btn-2")
                    .font(.system(size: 26))
                    .foregroundColor(Color(red: 0.05, green: 0.05, blue: 0.1).opacity(0.85))
                    .frame(width: 36.0, height: 36.0)
                    .padding(8.0)
                    .background(Color(red: 1.00, green: 1.00, blue: 0.80))
            }
            .background(Color(red: 0.11, green: 0.11, blue: 0.17))
        }
        .frame(minWidth: nil, maxWidth: .infinity, minHeight: nil, maxHeight: .infinity, alignment: .leading)
        .padding(12.0)
        .background(Color(red: 0.11, green: 0.11, blue: 0.17))
    }
}

File: testdata/tpl_nav_bar/expected.dart
Widget build(BuildContext context) {
  return   Container(
    width: double.infinity,
    height: 56.0,
    padding: EdgeInsets.all(12.0),
    margin: EdgeInsets.all(0.0),
    child:     Row(
      mainAxisAlignment: MainAxisAlignment.spaceBetween,
      crossAxisAlignment: CrossAxisAlignment.center,
      children: [
        Flexible(
          fit: FlexFit.loose,
          child:           Container(
            width: 48.0,
            height: 48.0,
            padding: EdgeInsets.all(8.0),
            margin: EdgeInsets.all(0.0),
            color: Color.fromRGBO(251, 180, 174, 1.0),
            alignment: Alignment.center,
            child: Text('logo',
              style: TextStyle(fontSize: 26, color: Color.fromRGBO(13, 13, 26, 0.85)),
            ),
          )
        ),
        Flexible(
          fit: FlexFit.loose,
          child:           Container(
            padding: EdgeInsets.all(0.0),
            margin: EdgeInsets.all(0.0),
            child:             Row(
              mainAxisAlignment: MainAxisAlignment.start,
              crossAxisAlignment: CrossAxisAlignment.center,
              children: [
                Flexible(
                  fit: FlexFit.loose,
                  child:                   Container(
                    width: 80.0,
                    height: 36.0,
                    padding: EdgeInsets.all(8.0),
                    margin: EdgeInsets.all(0.0),
                    color: Color.fromRGBO(179, 205, 227, 1.0),
                    alignment: Alignment.center,
                    child: Text('link-1',
                      style: TextStyle(fontSize: 26, color: Color.fromRGBO(13, 13, 26, 0.85)),
                    ),
                  )
                ),
                Flexible(
                  fit: FlexFit.loose,
                  child:                   Container(
                    width: 80.0,
                    height: 36.0,
                    padding: EdgeInsets.all(8.0),
                    margin: EdgeInsets.all(0.0),
                    color: Color.fromRGBO(204, 235, 197, 1.0),
                    alignment: Alignment.center,
                    child: Text('link-2',
                      style: TextStyle(fontSize: 26, color: Color.fromRGBO(13, 13, 26, 0.85)),
                    ),
                  )
                ),
                Flexible(
                  fit: FlexFit.loose,
                  child:                   Container(
                    width: 80.0,
                    height: 36.0,
                    padding: EdgeInsets.all(8.0),
                    margin: EdgeInsets.all(0.0),
                    color: Color.fromRGBO(222, 203, 228, 1.0),
                    alignment: Alignment.center,
                    child: Text('link-3',
                      style: TextStyle(fontSize: 26, color: Color.fromRGBO(13, 13, 26, 0.85)),
                    ),
                  )
                ),
              ],
            )
          )
        ),
        Flexible(
          fit: FlexFit.loose,
          child:           Container(
            padding: EdgeInsets.all(0.0),
            margin: EdgeInsets.all(0.0),
            child:             Row(
              mainAxisAlignment: MainAxisAlignment.start,
              crossAxisAlignment: CrossAxisAlignment.center,
              children: [
                Flexible(
                  fit: FlexFit.loose,
                  child:                   Container(
                    width: 36.0,
                    height: 36.0,
                    padding: EdgeInsets.all(8.0),
                    margin: EdgeInsets.all(0.0),
                    color: Color.fromRGBO(254, 217, 166, 1.0),
                    alignment: Alignment.center,
                    child: Text('btn-1',
                      style: TextStyle(fontSize: 26, color: Color.fromRGBO(13, 13, 26, 0.85)),
                    ),
                  )
                ),
                Flexible(
                  fit: FlexFit.loose,
                  child:                   Container(
                    width: 36.0,
                    height: 36.0,
                    padding: EdgeInsets.all(8.0),
                    margin: EdgeInsets.all(0.0),
                    color: Color.fromRGBO(255, 255, 204, 1.0),
                    alignment: Alignment.center,
                    child: Text('btn-2',
                      style: TextStyle(fontSize: 26, color: Color.fromRGBO(13, 13, 26, 0.85)),
                    ),
                  )
                ),
              ],
            )
          )
        ),
      ],
    )
  )
;
}
